lib/p2p/Pool.ts

```diff
@@ -117,6 +124,7 @@ class Pool extends EventEmitter {
     const peer = this.peers.get(socketAddress);
     if (peer) {
       peer.destroy();
+      this.logger.warn(`Disconnected from ${peer.pubKey} ${address} ${port}`);
```

Comment:
I don't think this needs to be a `warn` - `info` seems appropriate. Also I think there should be a `:` between `address` and `port` rather than a space.